tests/test_utils.py

```diff
@@ -0,0 +1,111 @@
+import pytest
+
+import ixmp
+import message_ix
+
+import pandas as pd
+import pandas.util.testing as pdt
+
+from numpy import testing as npt
+
+import message_ix.utils as utils
+from testing_utils import test_mp
+
+
+msg_args = ('canning problem (MESSAGE scheme)', 'standard')
+
+
+def test_add_spatial_single(test_mp):
+    scen = test_mp.Scenario(*msg_args, version='new', scheme='MESSAGE')
+    data = {'country': 'Austria'}
+    utils.add_spatial_sets(scen, data)
+
+    exp = ['World', 'Austria']
+    obs = scen.set('node')
+    npt.assert_array_equal(obs, exp)
+
+    exp = ['World', 'global', 'country']
```

Comment:
should `'World'` be really in here? I'd think that `World` is the node at the `global` level...

Reply:
I was surprised too, but this passes the tests =)